ska_conda/ska_builder.py

```diff
@@ -59,30 +81,20 @@ def _build_package(self, name):
         print("Building package %s." % name)
         pkg_path = os.path.join(pkg_defs_path, name)
         cmd_list = ["conda", "build", pkg_path, "--croot",
-                    self.ska_build_dir, "--no-test",
-                    "--no-anaconda-upload"]
+                    self.build_dir, "--no-test",
+                    "--no-anaconda-upload", "--skip-existing"]
```

Comment:
It looks like the --skip-existing option is smart enough that it skips if you have an existing build at the requested version (not just a package built with the name) so this seems to just work for our use cases.

Reply:
For verification of the behavior, I had just done a single test of this outside this code:

 - built Ska.Shell at 3.3.1
 - tried to build again with --skip-existing, but no build was done
 - checked out the repo at 3.3.2
 - tried to rebuild with --skip-existing and it built 3.3.2

Hopefully there are no gotchas.

Reply:
I think this is not going to help conda to "know" when any meta packages should be rebuilt, so if we want that automated, we will need to figure out a mechanism.